docs/source/release/v4.1.0/framework.rst

```diff
@@ -27,6 +27,7 @@ Improvements
 - :ref:`AbsorptionCorrection <algm-AbsorptionCorrection>` has a new property `ScatterFrom` which allows for calculating the correction for the other components (i.e. container and environment)
 - Prevent an error due to the locale settings which may appear when reading, for instance, the incident energy Ei value from the logs in :ref:`ConvertUnits <algm-ConvertUnits>` and many other algorithms.
 - :code:`indices` and :code:`slicepoint` options have been added to :ref:`mantid.plots <mantid.plots>` to allow selection of which plane to plot from an MDHistoWorkspace
+- :ref:`Pseudo-Voigt <func-PseudoVoigt>` has been modified for better physics meaning.
```

Comment:
A clearer meaning might be that "Pseudo-Voigt has been modified to be more in line with FULLPROF and GSAS.